truss/templates/server.Dockerfile.jinja

```diff
@@ -109,10 +109,13 @@ COPY ./{{ config.model_module_dir }} /app/model
 {% block run %}
     {%- if config.docker_server %}
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-        curl nginx python3-pip && \
-        rm -rf /var/lib/apt/lists/*
+        curl nginx && rm -rf /var/lib/apt/lists/*
```

Comment:
Removes the `python3-pip` installation, having `uv` manage that below is better (plus we get more control over the version)